WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -130,6 +135,20 @@ class NotificationsViewController : UITableViewController
         tableViewHandler.updateRowAnimation = .None
     }
 
+    override func viewDidDisappear(animated: Bool) {
+        super.viewDidDisappear(animated)
+
+        // Glitch Workaround. Scenario:
+        //  -   Mark as Read saves the mainMOC
+        //  -   Simperium sends the change
+        //  -   The backend acknowledges the change, and merges back into the mainMOC
+        // This causes a reloadData event, that might cut any ongoing animations in the detail views. For that reason,
```

Comment:
Can you expand this point a bit with more specific details?  Right now I understand what's going on but don't have any specific method references to check the chain of events leading to the `reloadData` call.

As a side note: I was trying to track that chain of events because I'd expect a save call not to refresh the local GUI.

A few thoughts:

- Maybe we can avoid the refresh call in the first place, since it's not needed?
- Does this VC need a live-update feature at all?  Do notifications change that much?